tests/unit_tests/gui/ertwidgets/test_caseselector.py

```diff
@@ -1,77 +0,0 @@
-import pytest
```

Comment:
Are the tests in this deleted file moved somewhere else or are they not needed anymore?

Reply:
Good catch, I accidentally removed it.